src/api.js

```diff
@@ -0,0 +1,11 @@
+const url = "https://bootcamp-api.codeit.kr";
+
+//todo: path로 prop으로 받아서 경로 나눠주기
+const getData = async ({ path = "" }) => {
+  const detailPath = path && `api/sample/${path}`;
+  const response = await fetch(`${url}/${detailPath}`);
+  const body = await response.json();
```

Comment:
이미 위에서 fetch 한 부분을 await 처리해줬기 때문에 이 부분은 await 가 의미가 없을 거 같아요!

```suggestion
  const body = response.json();
```